lawnchair/src/app/lawnchair/ui/preferences/IconPackPreferences.kt

```diff
@@ -156,12 +155,48 @@ fun IconPackPreferences() {
                     onValueChange = {
                         themedIconsAdapter.onChange(newValue = it.themedIcons)
                         drawerThemedIconsAdapter.onChange(newValue = it.drawerThemedIcons)
+                        iconPackAdapter.onChange(newValue = "")
+                        themedIconPackAdapter.onChange(newValue = "")
                     },
                     description = if (themedIconsAvailable.not()) {
                         stringResource(id = R.string.lawnicons_not_installed_description)
                     } else null,
                 )
             }
+
+            ExpandAndShrink(visible = themedIconsAdapter.state.value && !drawerThemedIconsAdapter.state.value) {
+                PreferenceGroup {
+                    PreferenceGroupHeading(heading = stringResource(id = R.string.themed_icon_pack))
```

Comment:
```suggestion
                PreferenceGroup (
                    heading = stringResource(id = R.string.themed_icon_pack)
                ) {
```

The same should be applied to other headings.